package.json

```diff
@@ -17,19 +14,19 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/sitebase/react-avatar.git"
+    "url": "https://github.com/QuickBase/react-avatar.git"
   },
   "author": "Wim Mostmans <wim@sitebase.be>",
```

Comment:
what ramification is there with keeping the same "author" and retaining the same MIT license? we may want to double check with Chris P about the correct process of doing this